tests/cecil-tests/Test.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+
+using NUnit.Framework;
+
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+
+#nullable enable
+
+namespace Cecil.Tests {
+
+	[TestFixture]
+	public class Test {
+
+		[TestCase ("/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/lib/32bits/Xamarin.iOS.dll")]
```

Comment:
This references the system install, not the locally built one.

You can compute the local path by including `tests/common/Configuration.cs` in the project, and then do something like this:

```csharp
var localPath = Path.Combine (Configuration.IOS_DESTDIR, assemblyName.SubString (1));
```

Adjust as required for Xamarin.Mac (MAC_DESTDIR).